migrations/versions/0438_request_invite_templates.py

```diff
@@ -0,0 +1,167 @@
+"""
+
+Revision ID: 0438_request_invite_templates
+Revises: 0437_min_numeric_scl_aux_tbls
+Create Date: 2023-12-11 16:21:05.947886
+
+"""
+import textwrap
+from alembic import op
+from flask import current_app
+
+
+revision = "0438_request_invite_templates"
+down_revision = "0437_min_numeric_scl_aux_tbls"
+
+
+request_invite_to_a_service_template_id = "77677459-f862-44ee-96d9-b8cb2323d407"
+request_invite_to_a_service_template_content = textwrap.dedent(
+    """\
```

Comment:
The slash is there after the first line to to avoid the empty line as suggested in the `textwrap` docs https://docs.python.org/3/library/textwrap.html#:~:text=For%20example%3A-,def,-test()%3A%0A%20%20%20%20%23%20end